src/libraries/Common/src/Interop/Interop.TimeZoneInfo.Stub.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+internal static partial class Interop
+{
+    internal static partial class Globalization
+    {
+        // needs to be kept in sync with TimeZoneDisplayNameType in System.Globalization.Native
+        internal enum TimeZoneDisplayNameType
```

Comment:
nit: we should keep this in `Interop.TimeZoneInfo.cs` and move the `GetTimeZoneDisplayName()` methods into `Interop.TimeZoneInfo.Browser.cs` and `Interop.TimeZoneInfo.UnixOrWindows.cs`